app/views/katello/api/v2/package_groups/index.json.rabl

```diff
@@ -4,4 +4,8 @@ extends "katello/api/v2/common/metadata"
 
 child @collection[:results] => :results do
   extends "katello/api/v2/package_groups/base"
+
+  if (params[:show_all_for] == "content_view_filter" && @filter)
+    node(:added_to_content_view_filter) { |package_group| package_group.in_content_view_filter?(@filter) }
```

Comment:
Would it be more flexible if we returned an array of the CV filter IDs which this package group belongs to ? Sorry if it's a bad question - still forming up my understanding of the change